ci/jenkins/README.md

```diff
@@ -29,6 +30,21 @@ should be deleted. This ensures that all tests are run on a clean testbed.
 * jenkins jobs validator: this job only executes for PRs that include changes to
   [ci/jenkins/jobs](/ci/jenkins/jobs). It validates the syntax of the jenkins jobs'
   configuration.
+* [EKS conformance/network policy](http://jenkins.antrea-ci.rocks/view/cloud/job/cloud-antrea-eks-conformance-test-native/)
+  community tests on EKS cluster using sonobuoy, focusing on "Conformance" and "Feature:NetworkPolicy", skipping the same regexes as in job __conformance__ above, as well as "NodePort" (See #690). <br>
```

Comment:
I am not sure the `<br>` tags will render properly on the netlify website, maybe you can use an empty line instead?

Reply:
Done